omnigibson/macros.py

```diff
@@ -13,11 +13,13 @@
 macros = Dict()
 gm = macros.globals
 
-# Paths (either relative to this directory or global absolute paths) for assets
+# Path (either relative to OmniGibson/omnigibson directory or global absolute path) for data
+gm.DATA_PATH = "data"
```

Comment:
there's a possibility that assets / dataset may be downloaded into different places, but it probably won't matter for now

Reply:
reverted change